bootstrap/013_task_queue.sql

```diff
@@ -0,0 +1,54 @@
+
+create or replace procedure task_queue.run_tasks()
+returns varchar
+language sql
+as
+begin
+show warehouses;
+show tasks in schema task_queue;
+let res resultset := (
+with tlist as (
+select "id" as id, "name" as name from table(result_scan(last_query_id())) where "name" <> 'run_tasks'
+),
+wlist as (
+select count(*) > 0 as wh_exists from table(result_scan(last_query_id(-2))) where "name" = 'SUNDECK_TASK_QUEUE_WH'
+),
+thist as (
+    select tlist.name, state from tlist left outer join account_usage.task_history th on tlist.id = th.task_id
+), tstate as (
+    select name, state, row_number() over (partition by name order by state desc) as rn from thist
```

Comment:
I'm a bit confused by this partitioning -- `order by state desc` seems odd. Is that what you intended?

It looks like it would be returning the oldest record for each task from the lexicographically-largest state (often, `SUCCEEDED`)